client/src/components/page/DirectionsButton.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { makeStyles } from "@material-ui/core/styles";
+import Button from "@material-ui/core/Button";
+import buttonMapImage from '../../assets/images/Google_Maps_icon.svg';
+
+type DirectionsProps = {
+  destination: string;
+}
+
+const useStyles = makeStyles({
+  buttonStuffContainer: {
+    display: 'flex',
+    flexDirection: 'row',
+    justifyContent: 'center',
+    alignItems: 'center',
+    textShadow: '0 0 1px white, 0 0 1px white, 0 0 1px white',
+  },
+  buttonText: {
+    textAlign: 'center',
+    lineHeight: '125%',
+
+  },
+  buttonImage: {
+    order: 0,
+    width: '4em'
+
+  },
+})
+
+let origin = "";
+
+function mapURL(destination : string): string {
```

Comment:
shouldn't you have a parameter `origin` passed in as well that you are using for the url?